### DISCUSSION TITLE
Add ability to access Slimproto settings

### DISCUSSION THREAD

**Discussion by Slim1399** [music-assistant/support#3223] [Feature requests and ideas]
Hi,
Why do we not have access to the local Slimproto interface of player provider ? This is necessary to adjust the latency of Bluetooth speakers synchronized with wired systems.

**Comment by OzGav** [comment]
Lack of time. I can change this to a feature request unless you are able to submit a PR.

**Reply by Slim1399** [reply]
If i could i had already done it. Sorry !
